extensions/3DTILES_implicit_tiling/README.md

```diff
@@ -0,0 +1,631 @@
+
+* **Type**: `number` `[6]`
+* **Required**: No
+
+| Region | Quadtree | Octree |
+|:---:|:--:|:--:|
+| ![](figures/region.png) | ![](figures/region_quadtree.png) | ![](figures/region_octree.png)  |
+
+#### boundingVolume.cell
+
+An array of 10 numbers that defines a geodesic quadrilateral cell. The first 8 elements are 4 pairs of EPSG:4979 coordinates in degrees in counterclockwise order, and the last 2 elements are heights in meters above the WGS84 ellipsoid.
+
+* **Type**: `number` `[10]`
+* **Required**: No
+
+| Cell | Quadtree | Octree |
+|:---:|:--:|:--:|
+| ![](figures/cell.png) | ![](figures/cell_quadtree.png) | ![](figures/cell_octree.png)  |
+
+
+---
+
+### subdivision
+
+Provides information about the subdivision of the tileset.
+
+**Properties**
+
+|   |Type|Description|Required|
+|---|----|-----------|--------|
+|**completeLevels**|`number`|An integer describing how many levels subdivide internally. We can assume the subdivision bitcode for all tiles on levels before this one to be `11`.|No|
+|**bufferView**|`number`|An index to the buffer view containing the subdivision buffer.|No|
+
+---
+
+### content
+
+Provides information about the content of the tileset.
+
+**Properties**
+
+|   |Type|Description|Required|
+|---|----|-----------|--------|
+|**levelOffset**|`number`|An integer describing which level the content bitsream, if provided, starts at|No|
+|**levelOffsetFill**|`number`|An integer describing the state value for the content bit for all levels up to the `levelOffset`.|No|
+|**bufferView**|`number`|An index to the buffer view containing the content buffer.|No|
+
+---
+
+### metadata
+
+Provides information about the metadata of the tileset.
+
+**Properties**
+
+|   |Type|Description|Required|
+|---|----|-----------|--------|
+|**levelOffset**|`number`|An integer describing which level the metadata bitsream, if provided, starts at|No|
+|**levelOffsetFill**|`number`|An integer describing the state value for the metadata bit for all levels up to the `levelOffset`.|No|
+|**bufferView**|`number`|An index to the buffer view containing the metadata buffer.|No|
+
+## Appendix
+
+### Algorithms
+
+#### Jump Buffer Construction
+
+```javascript
+function createJumpBuffer(sBuffer) {   
+    let jBuffer = [];
+    let sBufferIndex = 1;
+    let currentLevel = 0;
+    while (sBufferIndex != sBuffer.length) {
+        currentLevel++;
+        jBuffer.push([]);
+        // Get maximum number of elements in this level, based on number of tiles in the level above that will subdivide.
+        let currentLevelMax = jBuffer[currentLevel - 1].length * tilingScheme;
+        jumpBufferSize += currentLevelMax;
+        for (let i = 0; i < currentLevelMax; i++) {
+            if (sBuffer[sBufferIndex++] === 1) {
+                jBuffer[currentLevel].push(jBuffer[currentLevel - 1][Math.floor(i / tilingScheme)] * tilingScheme + (i % tilingScheme));
+            }
+        }
+    }
+    return jBuffer;
+}
+```
+
+#### Tile Index Lookup
```

Comment:
Is there a fast path for complete trees? Would be good to show off true random access.

Reply:
Added a note about that, but next draft will have a more fleshed out algorithm.